### ISSUE TITLE
qat

### ISSUE THREAD

**Issue by ZhaoyangJoker** [intel/QAT_Engine#230]

**Comment by Yogaraj-Alamenda** [comment]
I believe you are using QAT_SW. It  so, yes ECDSA verify is not supported in QAT_SW and will use default OpenSSL implementation

**Comment by ZhaoyangJoker** [comment]
I also test nginx + qat_sw + ecc256, the speed is same with and without qat。The reason is that ECDSA verify is not supported in QAT_SW？

**Comment by Yogaraj-Alamenda** [comment]
Yes it supports sign and performance with ecdsa p256 is not that great compared to other algorithms although there is some gain. Can you share your performance comparison? Other curve p384 is much faster compared to without qat than p256.

**Comment by ZhaoyangJoker** [comment]
Hi @Yogaraj-Alamenda Thank you for answering my previous questions.
Here are my test data：
8 workers nginx , with qat ops is 8939，CPU usage is 55%~68%; nginx without qat ops is 9298 ,CPU usage is 58%~68%.
I don't know why nginx without qat is quicker than nginx with qat.
